src/pybind/mgr/dashboard/frontend/cypress/integration/orchestrator/workflow/06-cluster-check.e2e-spec.ts

```diff
@@ -30,6 +32,25 @@ describe('when cluster creation is completed', () => {
       hosts.navigateTo();
     });
 
+    // avoid creating node-exporter on the newly added host
+    // to favour the host draining process
+    it('should reduce the count for node-exporter', () => {
+      services.editService('node-exporter', '3');
+    });
+
+    // grafana ip address is set to the fqdn by default.
+    // kcli is not working with that, so setting the IP manually.
+    it('should change ip address of grafana', { retries: 2 }, () => {
+      const dashboardArr: Input[] = [
+        {
+          id: 'GRAFANA_API_URL',
+          newValue: 'https://192.168.100.100:3000',
```

Comment:
@epuertat @aaSharma14 recently some changes went into cephadm which basically means fetching the fqdn for some config urls. And our kcli configuration causes the fqdn to be something like `https://ceph-node-00.ceph-dashboard:3000` and this doesn't work. So for now I had to manually change the grafana-api-url.